README.md

```diff
@@ -8,7 +8,7 @@ This is inspired by [ko](https://github.com/google/ko) which is awesome but does
 
 ```bash
 mkctr \
-  --base="ghcr.io/tailscale/alpine-base:3.14" \
+  --base="ghcr.io/tailscale/alpine-base:3.16" \
```

Comment:
If we own this as well, should we just document `latest`?

Reply:
we don't push latest tag for this image

Reply:
This is just an example, we can replace it with an upstream alpine image